android/guava-tests/test/com/google/common/util/concurrent/AbstractFutureTest.java

```diff
@@ -432,6 +432,8 @@ public void testFutureBash() {
                 + 50 // for the listeners
                 + 50 // for the blocking get threads,
                 + 1); // for the main thread
+    // Refactoring this causes AbstractFutureFallbackAtomicHelperTest to hang forever
+    // (https://github.com/ponder-lab/guava/actions/runs/9654663319/job/26629281352#step:6:1102)
```

Comment:
Refactoring this causes [AbstractFutureFallbackAtomicHelperTest to hang forever](https://github.com/ponder-lab/guava/actions/runs/9654663319/job/26629281352#step:6:1102)